en/companion_computer/holybro_pixhawk_rpi_cm4_baseboard.md

```diff
@@ -166,7 +297,7 @@ eth0    Link encap:Ethernet HWaddr xx:xx:xx:xx:xx:xx at UP
 ```
 
 However, it doesn’t have an IP yet.
-Set one that is similar to the one on the RasPi CM4:
+Set one that is similar to the one on the RPi CM4:
```

Comment:
FYI RasPi is far more common in "the world" than RPi. However I have moved to your version for this doc to make description consistent.